Modules/termios.c

```diff
@@ -27,6 +27,9 @@
 
 #include <termios.h>
 #include <sys/ioctl.h>
+#if defined(__sun) && defined(__SVR4)
+#  include <unistd.h>     // ioctl
```

Comment:
```suggestion
#  include <unistd.h>             // ioctl()
```

Reply:
It's just a coding convention used in other C files.

Reply:
I see, I wasn't sure how many spaces should I put there (and missed those parentheses).